src/test/java/ezvcard/io/roundtrip/RoundTripTestBase.java

```diff
@@ -0,0 +1,219 @@
+package ezvcard.io.roundtrip;
+
+import static org.junit.Assert.*;
+
+import java.io.*;
+import java.util.List;
+
+import ezvcard.VCard;
+import ezvcard.VCardVersion;
+import ezvcard.io.StreamReader;
+import ezvcard.io.StreamWriter;
+import ezvcard.io.text.VCardReader;
+import ezvcard.io.text.VCardWriter;
+import ezvcard.property.VCardProperty;
+import ezvcard.util.IOUtils;
+
+public abstract class RoundTripTestBase {
+
+	static final class Filter implements FilenameFilter {
+		private final String extension;
+		private final String[] excludes;
+
+		private Filter(String extension, String... excludes) {
+			this.extension = "." + extension;
+			if (excludes != null) {
```

Comment:
This null check is not necessary.  If no vararg parameters are passed in, then the "excludes" variable will be an empty array.